src/sage/rings/polynomial/infinite_polynomial_ring.py

```diff
@@ -884,6 +884,17 @@ def _element_constructor_(self, x):
             Traceback (most recent call last):
             ...
             ValueError: cannot convert 1/3 into an element of Infinite polynomial ring in x over Integer Ring
+
+        .. WARNING::
+
+            The :issue:`37756` is not yet fixed::
+
+                sage: L.<x, y> = QQ[]
+            sage: R.<a> = InfinitePolynomialRing(QQ)
+            sage: M = InfinitePolynomialRing(L, names=["a"])
+            sage: c = a[0]
+            sage: M(c)  # known bug
+            a_0
```

Comment:
```suggestion
                sage: R.<a> = InfinitePolynomialRing(QQ)
                sage: M = InfinitePolynomialRing(L, names=["a"])
                sage: c = a[0]
                sage: M(c)  # known bug
                a_0
```
The rest of these need to be adjusted too; sorry I wasn't explicit about it.

Reply:
I'm sorry, that was my fault.